docs/user/document/tracking/aliases-usage.en.md

```diff
@@ -1,8 +1,10 @@
 # Aliases - Usage
 
-Aliases can be used to indicate the number assigned to this advisory by the coordinator or manufacturer.
+Aliases can be used to indicate the number, identifier or name assigned to this advisory, case or issue by the coordinator, manufacturer, supplier or other party. This can also include internal ticket numbers.
 
 *Examples:*
 
 * ICS Cert ID: `ICSA-20-168-01`
-* Cert CC: `VU#257161` (Ripple20)
+* CERT/CC: `VU#257161` (Ripple20)
+* AMNESIA:33
+* OT:ICEFALL
```

Comment:
As you used the verbatim font for the others - maybe we should do the same here? As well as: Maybe we should also remove the "CERT/CC" and "ICS-CERT ID" as those should not be a part of the value... :thinking: 

```suggestion
* `ICSA-20-168-01`
* `VU#257161` 
* `Ripple20`
* `AMNESIA:33`
* `OT:ICEFALL`

Reply:
updated